src/twisted/conch/ssh/userauth.py

```diff
@@ -306,6 +318,23 @@ def auth_publickey(self, packet: bytes) -> Deferred[_ConchPortalTuple]:
             )
         return result
 
+    # Follows the OpenSSH SK signature format specified at
+    # https://github.com/openssh/openssh-portable/blob/a4aa090a3d40dddb07d5ebebc501f6457541a501/PROTOCOL.u2f#L176
+    @staticmethod
+    def _wrapSecurityKeySignedData(signature, originalSignedData, application):
+        _, _, trailing = getNS(signature, 2)
```

Comment:
Is it ok to ignore the key type ?

Also, why are we ignoring the signature ?

Maybe fail we got an unexpected value here.
```suggestion
        # We ignore the key type as the value is present in the
        # to be signed message. So we validate it in that way.
        _, _, trailing = getNS(u2fSignature, 2)
```

Reply:
Yes, I think it is okay to ignore the key type here. This blob is later verified using the signature. I've updated the function name and docstring to better reflect the purpose